apps/src/code-studio/labRedux.js

```diff
@@ -0,0 +1,30 @@
+const SET_IS_LOADING = 'lab/SET_IS_LOADING';
```

Comment:
can we write this using redux-toolkit and typescript? [instructions doc](https://docs.google.com/document/d/1EYM4tbOnMldGVGj6AtrNkRT3AKxuZZiLXuAL9fjR5VM/edit)

Reply:
Also should we put this under `/labs`?

Reply:
Done and done.